sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/command.py

```diff
@@ -72,7 +73,7 @@ def resources(self, value: JobResourceConfiguration):
 
     @classmethod
     def _picked_fields_from_dict_to_rest_object(cls) -> List[str]:
-        return ["environment", "limits", "resources"]
+        return ["environment", "limits", "resources", "environment_variables"]
```

Comment:
Do we need to modify the `from_rest` method to make sure node entity has env variable when load from rest?

Reply:
what do you mean by modifying `from_rest`?
if returned rest object has an environment_variable, it will be passed to `__init__` by default
if returned rest object has not environment_variable, environment_variable will be set as `None` in `__init__`